examples/named_entity_recognition/Named_Entity_Recognition_with_Pytorch_and_🔭_Galileo.ipynb

```diff
@@ -23,6 +23,13 @@
       "outputs": [],
```

Comment:
<p>I think we should look into trying to hide that better and not have it show up in the training loop. I'm worried that will confuse data scientists, especially that most of us see CI testing like that for the first time! Wdyt ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rungalileo/examples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>